src/Components/Publishing/Constants.ts

```diff
@@ -6,4 +6,6 @@ export const zIndex = {
 
 export const articleHref = slug => `/article/${slug}`
 
+export const articleFullHref = slug => `https://www.artsy.net/article/${slug}`
```

Comment:
I think ideally we want to replace this with a configurable `baseUrl` from somewhere like `sharify` in Force but there's no nice way to pass this down from the main <Article/> component without having to go through a long tree or use `context`. Yet another reason to move to a better mgmt container like Redux!

Reply:
Could this be based on `window.location` instead?

Reply:
If you do ever want to add some context, you can do so here if it’s something that could be useful data for many components https://github.com/artsy/reaction/blob/master/src/Components/Artsy.tsx

Reply:
Oh cool, I'll take a deeper look at this context work. I think `window.location` could become a pain later on if we decide to do SSR-only stuff like Google AMP.

Reply:
Gotcha, makes sense 👍